scripts/utilities/retry.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+
+/**
+ * Retry - Retry a function on failure
+ *
+ * @param {function} Run          - The function to retry on failure
+ * @param {string}   errorMessage - Additional information when the limit is reached
+ * @param {number}   attempts     - The total number of attempts
+ * @param {number}   limit        - The maximum number of attempts
+ * @param {number}   delay        - The seconds to wait between attempts
+ */
+const Retry = function(Run, errorMessage, attempts = 0, limit = 3, delay = 10) {
+  // Check that there is a function to try
+  if (typeof Run !== 'function') {
+    console.error('Retry must have a function to run and retry on failure');
+    return;
+  }
+
+  let currentAttempts = attempts;
```

Comment:
This assignment doesn't really make sense to me, why can't you just use the function parameter the whole way through?

Reply:
I had an odd linting error, i'll fix it up!